Data/Sys/GameSettings/GGTE01.ini

```
@@ -17,3 +17,26 @@ $Max/Infinite Money
 $Max Happy Points
 0728E81C 08000000
 0438F73C 0001869F
+$16:9 Widescreen [darkludx]
+0416C4B0 C3A29F60
+0465DCA0 3FE38E39
+
+[AR_RetroAchievements_Verified]
+$16:9 Widescreen [darkludx]
+
+[Gecko]
+$Simple 60FPS [Minty Meeo]
+0400a518 38600001
+* Requires Emulated CPU Clock Override
+$Experimental 60fps [Minty Meeo]
+C20157D4 00000004
+3C608036 8063B7A4
+2C030000 38600001
+41A20008 38600002
+60000000 00000000
+* Requires Emulated CPU Clock Override
+* Requires Speed Up Disc Transfer Rate
```

The setting has been reworked since this note was written.

```suggestion
* Requires disabling Emulate Disc Speed
```